src/v/transform/tests/test_fixture.cc

```diff
@@ -72,9 +72,21 @@ ss::future<> fake_source::push_batch(model::record_batch batch) {
     co_return;
 }
 
-ss::future<> fake_wasm_engine::start() { return ss::now(); }
+ss::future<> fake_wasm_engine::start() {
+    if (_started) {
+        throw std::logic_error("starting already started wasm engine");
```

Comment:
Don't see a test for this. Did you intend to add one, or was this added "for completeness"?

Reply:
Yeah it's for completeness (and to catch and accidental bugs).